dev/io.openliberty.wsoc.internal_fat/bnd.bnd

```diff
@@ -0,0 +1,45 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src:\
+    fat/src,\
+    test-applications/basic.jar/src,\
+    test-applications/basic.war/src,\
+	test-applications/websocket11.war/src
+
+fat.project: true
+
+# Define additional tested features that are NOT present any any XML files in this bucket.
+#tested.features:\
+
+
+-buildpath: \
+    com.ibm.ws.junit.extensions;version=latest,\
+    com.ibm.ws.common.encoder;version=latest,\
+    com.ibm.websphere.javaee.ejb.3.1;version=latest,\
+    com.ibm.websphere.javaee.cdi.1.0;version=latest,\
+    com.ibm.websphere.javaee.servlet.3.1;version=latest,\
+    com.ibm.websphere.javaee.el.3.0;version=latest,\
+    com.ibm.websphere.javaee.jsp.2.3;version=latest,\
+    com.ibm.ws.jsp.2.3;version=latest,\
```

Comment:
do we actually need jsp/el/servlet here?

Reply:
I was basing myself on what was included in the classpath of the wsoc test in CL. I tried leaving them out but got some errors about class not found exceptions. Ill try again to see if I run into the same errors